core/float/truncate_spec.rb

```diff
@@ -3,4 +3,14 @@
 
 describe "Float#truncate" do
   it_behaves_like(:float_to_i, :truncate)
+
+  ruby_version_is "2.4" do
+    it "returns self to an optionally given precision" do
```

Comment:
I think we are missing a `truncated` between `returns self` and `to an optionally given precision`